static/app/views/replays/detail/memoryChart.tsx

```diff
@@ -56,7 +56,9 @@ function MemoryChart({
       right: space(1),
     }),
     tooltip: Tooltip({
+      appendToBody: true,
       trigger: 'axis',
+      renderMode: 'html',
```

Comment:
what does this do?

Reply:
From the type def for the tooltips says the renderMode should be html if using appendToBody

```
    /**
     * 'auto': use html by default, and use non-html if `document` is not defined
     * 'html': use html for tooltip
     * 'richText': use canvas, svg, and etc. for tooltip
     */
    renderMode?: 'auto' | TooltipRenderMode;
    /**
     * If append popup dom to document.body
     * Only available when renderMode is html
     */
    appendToBody?: boolean;
    /**
     * specified class name of tooltip dom
     * Only available when renderMode is html
     */
    className?: string;
    ```